lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -273,9 +273,6 @@ def invalid_v1_location(package)
         def invalid_location_uri_error(package, location_options)
           path = location_options.url
           return nil if path == ZendeskAppsSupport::Manifest::LEGACY_URI_STUB
-          if path.include?('{{setting.')
-            return location_options.signed ? ValidationError.new(:signed_setting_uri, uri: path) : nil
```

Comment:
can remove [this string](https://github.com/zendesk/zendesk_apps_support/blob/master/config/locales/en.yml#L83) too?

Reply:
And [this string](https://github.com/zendesk/zendesk_apps_support/blob/917e290f5d7a4727122f1aa0e095658bcc77c8f9/config/locales/translations/zendesk_apps_support.yml#L193) too

Reply:
Removed it locally but forgot to push this one out. Done now.

Btw, when do we remove this translation key? It won't be used, i guess i can remove it along with this change-set?
https://github.com/zendesk/zendesk_apps_support/blob/917e290f5d7a4727122f1aa0e095658bcc77c8f9/config/locales/en.yml#L83